nimbus-jose-jwt_aws-kms-extension/src/test/java/com/nimbusds/jose/aws/kms/crypto/utils/JWEHeaderUtilTest.java

```diff
@@ -118,10 +124,62 @@ void beforeEach() {
                 @DisplayName("shouldn't throw any exception.")
                 void shouldThrowException() {
                     assertThatNoException()
-                            .isThrownBy(() -> JWEHeaderValidationUtil.validateJWEHeaderAlgorithms(
+                            .isThrownBy(() -> JWEHeaderUtil.validateJWEHeaderAlgorithms(
                                     testJweHeader, testSupportedAlgorithms, testSupportedEncryptionMethods));
                 }
             }
         }
     }
+
+    @Nested
+    @DisplayName("the getJWEHeaderWithEncryptionContext method,")
+    class GetJWEHeaderWithEncryptionContextMethod {
+
+        private JWEHeader testHeader;
+
+        private String tesEncryptionContextHeaderName;
```

Comment:
[TYPO] tes -> test

Reply:
Resolve in the next commit. Please take a look.